app/components/dropdown-search.js

```diff
@@ -0,0 +1,134 @@
+import Ember from "ember";
+import { singularize, pluralize } from "ember-inflector";
+import _ from "lodash";
+const { getOwner } = Ember;
+
+export default Ember.Component.extend({
+  selected: [],
+  previousValue: "",
+  store: Ember.inject.service(),
+  subformDetailService: Ember.inject.service(),
+  resourceType: Ember.computed.alias("packageDetails"),
+  fixedDropdownArr: ["frequency", "voltage", "compTestStatus", "testStatus"],
+  selectedData: Ember.computed.alias("selectedValue"),
+  selectedDataDisplay: Ember.computed.alias("selectedValuesDisplay"),
+  fixedDropdownArr: ["frequency", "voltage", "compTestStatus", "testStatus"],
+  fixedDropdownArrId: [
+    "frequency_id",
+    "voltage_id",
+    "compTestStatus_id",
+    "testStatus_id"
+  ],
+
+  displayLabel: Ember.computed("addAble", function() {
+    return this.get("addAble") ? "Add New Item" : "";
+  }),
+
+  selectedOptionDisplay: Ember.computed("dropDownValues", function() {
```

Comment:
I am not able to understand the names here
`selectedOptionDisplay` and  `selectedDataDisplay`.. It's very confusing